stablehlo/dialect/StablehloOps.cpp

```diff
@@ -2171,12 +2171,12 @@ void SortOp::build(OpBuilder& builder, OperationState& state,
   state.addRegion();
 }
 
-LogicalResult SortOp::inferReturnTypeComponents(
-    MLIRContext*, Optional<Location> location, ValueShapeRange operands,
+LogicalResult SortOp::inferReturnTypes(
```

Comment:
What is the reason for refactoring to `inferReturnTypes`? I think that `inferReturnTypeComponents` recently added support for encodings as well.

Reply:
+1. The "recently added support" Eugene mentioned is here: https://github.com/llvm/llvm-project/commit/b2505ca2ece3
In short, in old days the PRs like this have to change to use `inferReturnTypes` but today it is not necessary any more